From 907de931f547bd0d6178ee04bdb6496cd66dbdd7 Mon Sep 17 00:00:00 2001
From: Tom Tromey <tromey@adacore.com>
Date: Fri, 3 Jun 2022 10:35:30 -0600
Subject: [PATCH] Use bool for evregpy_no_listeners_p

I noticed that evregpy_no_listeners_p should return a bool.  This
patch makes this change.  I'm checking it in.
---
 gdb/python/py-events.h      | 2 +-
 gdb/python/py-evtregistry.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gdb/python/py-events.h b/gdb/python/py-events.h
index bd4a75c..1680b58 100644
--- a/gdb/python/py-events.h
+++ b/gdb/python/py-events.h
@@ -52,6 +52,6 @@ struct events_object
 extern events_object gdb_py_events;
 
 extern eventregistry_object *create_eventregistry_object (void);
-extern int evregpy_no_listeners_p (eventregistry_object *registry);
+extern bool evregpy_no_listeners_p (eventregistry_object *registry);
 
 #endif /* PYTHON_PY_EVENTS_H */
diff --git a/gdb/python/py-evtregistry.c b/gdb/python/py-evtregistry.c
index 003fe44..ef96c48 100644
--- a/gdb/python/py-evtregistry.c
+++ b/gdb/python/py-evtregistry.c
@@ -115,7 +115,7 @@ gdbpy_initialize_eventregistry (void)
 /* Return the number of listeners currently connected to this
    registry.  */
 
-int
+bool
 evregpy_no_listeners_p (eventregistry_object *registry)
 {
   return PyList_Size (registry->callbacks) == 0;
-- 
2.33.0

